test/parallel/test-crypto-keygen.js

```diff
@@ -16,7 +16,7 @@ const {
   sign,
   verify
 } = require('crypto');
-const { promisify } = require('util');
+const { promisify, inspect } = require('util');
```

Comment:
Micro-nit, but I prefer these lists be kept in alphabetical order. Doesn't matter much, until they start getting long.

```suggestion
const { inspect, promisify } = require('util');
```